src/winter_onboarding_2021/core.clj

```diff
@@ -11,4 +11,16 @@
 
 (print (str 1 2 3))
 
-(print (+ 1 3) (+ 1 2 3) 4)
\ No newline at end of file
+(print (+ 1 3) (+ 1 2 3) 4)
```

Comment:
This is a bad commit message. It doesn't give any context about the contents of the commit. 
"tasks done on 6th oct" has no meaning by itself. If a few weeks, you would yourself forget what task was done on 6th oct.